resources/js/processes/modeler/components/ModelerApp.vue

```diff
@@ -93,13 +100,24 @@
       });
       return notifications;
     },
-    saveBpmn({xml, svg}) {
+    actionSaveBpmn ({xml, svg}) {
+      this.dataXmlSvg.xml = xml;
+      this.dataXmlSvg.svg = svg;
+
+      this.externalEmit.forEach(item => {
+        window.ProcessMaker.EventBus.$emit(item);
+      })
+      if (this.saveEmitDefault) {
+        window.ProcessMaker.EventBus.$emit('modeler-save');
+      }
+    },
+    saveBpmn(resolve, reject) {
```

Comment:
This isn't a promise - these are `onSuccess` and `onError` handlers that can optionally be passed in.

Could we change the naming from `resolve` to `onSuccess` and `reject` to `onError` since using the promise terminology misinforms?